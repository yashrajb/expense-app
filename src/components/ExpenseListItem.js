import React from 'react';
import { Link } from 'react-router-dom';


const ExpenseListItem = ({ dispatch, id, description, amount, createdAt }) => (
  <div>
    <h3><Link to={`/edit/${id}`}>{description}</Link></h3>
    <p>{amount} - {createdAt}</p>
  </div>
);

export default ExpenseListItem;
