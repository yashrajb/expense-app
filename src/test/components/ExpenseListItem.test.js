import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixture/commonData';
import ExpenseListItem from '../../components/ExpenseListItem';


test('should render expenseListitem',() => {
    
        const result = shallow(<ExpenseListItem {...expenses[0]}/>);
        expect(result).toMatchSnapshot();
    
    });