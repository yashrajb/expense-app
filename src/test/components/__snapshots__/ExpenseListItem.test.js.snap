// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenseListitem 1`] = `
<div>
  <h3>
    <Link
      replace={false}
      to="/edit/123456789"
    >
      Gum
    </Link>
  </h3>
  <p>
    195
     - 
    0
  </p>
</div>
`;
